xtask/src/docgen.rs

```diff
@@ -48,6 +50,22 @@ pub fn typable_commands() -> Result<String, DynError> {
     Ok(md)
 }
 
+pub fn static_commands() -> Result<String, DynError> {
+    let mut md: String = String::new();
+    md.push_str(&md_table_heading(&[
+        "Name".to_owned(),
+        "Description".to_owned(),
+    ]));
+
+    let cmdify = |s: &str| format!("`{}`", s);
+
+    for cmd in MappableCommand::STATIC_COMMAND_LIST {
+        md.push_str(&md_table_row(&[cmdify(cmd.name()).to_owned(), cmd.doc().to_owned()]));
```

Comment:
```suggestion
        md.push_str(&md_table_row(&[md_mono(cmd.name()), cmd.doc().to_owned()]));
```

we can reuse `md_mono` and eliminate `cmdify`